packages/front-end/components/Features/SDKConnections/SDKConnectionForm.tsx

```diff
@@ -121,22 +141,18 @@ export default function SDKConnectionForm({
       ? "backend"
       : "hybrid";
 
-  const selectedLanguagesWithoutRemoteEvalSupport = languages.filter(
-    (l) => !languageMapping[l].supportsRemoteEval
-  );
-
-  const selectedLanguagesWithoutEncryptionSupport = languages.filter(
-    (l) => !languageMapping[l].supportsEncryption
+  const maxSdkCapabilities = getConnectionSDKCapabilities(
```

Comment:
Not sure if I understand what max capabilities means here. Is it max capabilities for the user's SDK? If so what is the difference between this and `currentSDKCapabilities`? I'll take a look at the actual function after this but maybe worth making the name of the consts more clear

Reply:
"max" is basically the latest SDK version's list of capabilities, whereas "current" are those present in the user's current SDK version